specification/computeschedule/ComputeSchedule.Management/examples/2024-10-01/ScheduledActions_VirtualMachinesCancelOperations.json

```diff
@@ -21,23 +21,24 @@
             "errorCode": "lmpnizhpzc",
```

Comment:
errorcode and errordetails here should be null

Reply:
is this done? I still see text for erroCode and details

Reply:
Thanks, it was a miss